src/main/java/org/mule/extension/introspection/ExtensionModel.java

```diff
@@ -36,7 +36,7 @@
  *
  * @since 1.0
  */
-public interface ExtensionModel extends Described, Capable
+public interface ExtensionModel extends Described, Capable, ExtensibleModel
```

Comment:
It can be confusing to have extensionModel and extensibleModel. Can we find a better name for ExtensibleModel?